### PR TITLE
Add Helm Chart repositories add action on Add page

### PR DESCRIPTION
**Pull request by vikram-raj**
**Story:** https://issues.redhat.com/browse/ODC-6635

**Description:** 
Add the Project Helm Chart repositories add card on the Aad page

**Screen shots:**
image

Kapture 2022-04-22 at 12 36 15

### CODE REVIEW

**Comment by debsmita1** [comment]
The `Cancel` button on the form doesn't work

**Comment by vikram-raj** [comment]
> The `Cancel` button on the form doesn't work

@debsmita1 should it will take back to Add page?

File: frontend/packages/helm-plugin/console-extensions.json [hunk 1]
```diff
@@ -119,6 +127,20 @@
...
     }
   },
+  {
+    "type": "dev-console.add/action",
+    "properties": {
+      "id": "project-helm-chart-repositories",
+      "groupId": "helm-chart-repositories",
+      "href": "/ns/:namespace/projecthelmchartrepositories/~new",
+      "label": "%helm-plugin~Helm Chart repositories%",
+      "description": "%helm-plugin~Add a Helm Chart Respository in the namespace%",
```
Comment (karthikjeeyar) [inline frontend/packages/helm-plugin/console-extensions.json:137]: shouldn't this be ` Helm Chart repository` ? We use lowercase in the project helm chart form's helptext. cc: @debsmita1 
 
```suggestion
      "description": "%helm-plugin~Add a Helm Chart repository in the namespace%",
```

File: frontend/packages/helm-plugin/console-extensions.json [hunk 2]
```diff
@@ -105,6 +105,14 @@
       "component": { "$codeRef": "helmComponents.ProjectHelmChartRepositoryForm" }
     }
   },
+  {
+    "type": "dev-console.add/action-group",
+    "properties": {
+      "id": "helm-chart-repositories",
+      "name": "%helm-plugin~Helm Chart repositories%",
```
Comment (debsmita1) [inline frontend/packages/helm-plugin/console-extensions.json:112]:
```suggestion
      "name": "%helm-plugin~Project Helm Chart repositories%",
```

Comment (vikram-raj) [reply frontend/packages/helm-plugin/console-extensions.json]: Updated

File: frontend/packages/helm-plugin/locales/en/helm-plugin.json [hunk 3]
```diff
@@ -3,8 +3,10 @@
   "Helm Chart Repositories": "Helm Chart Repositories",
   "Project Helm Chart Repository": "Project Helm Chart Repository",
   "Project Helm Chart Repositories": "Project Helm Chart Repositories",
+  "Project Helm Chart repositories": "Project Helm Chart repositories",
   "Helm Chart": "Helm Chart",
   "Browse the catalog to discover and install Helm Charts": "Browse the catalog to discover and install Helm Charts",
+  "Add a Helm Chart respository in the namespace": "Add a Helm Chart respository in the namespace",
```
Comment (karthikjeeyar) [inline frontend/packages/helm-plugin/locales/en/helm-plugin.json:9]:
```suggestion
  "Add a Helm Chart repository in the namespace": "Add a Helm Chart respository in the namespace",
```

**Comment by karthikjeeyar** [comment]
Verified on cluster, it works as expected

/lgtm

**Comment by debsmita1** [comment]
Propagating docs and px acks from the Epic
/label docs-approved
/label px-approved

**Comment by kjack-redhat** [comment]
Looks good to me.

**Comment by debsmita1** [comment]
@vikram-raj The namespace dropdown is missing from the form

**Comment by vikram-raj** [comment]
> @vikram-raj The namespace dropdown is missing from the form

There is no namespace field in the form [CreateProjectHelmChartRepositoryForm.tsx](https://github.com/openshift/console/blob/master/frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/CreateProjectHelmChartRepositoryForm.tsx)

**Comment by debsmita1** [comment]
> > @vikram-raj The namespace dropdown is missing from the form
> 
> There is no namespace field in the form [CreateProjectHelmChartRepositoryForm.tsx](https://github.com/openshift/console/blob/master/frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/CreateProjectHelmChartRepositoryForm.tsx)

Sorry, I meant the namespace bar

**Comment by vikram-raj** [comment]
> > > @vikram-raj The namespace dropdown is missing from the form
> > 
> > 
> > There is no namespace field in the form [CreateProjectHelmChartRepositoryForm.tsx](https://github.com/openshift/console/blob/master/frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/CreateProjectHelmChartRepositoryForm.tsx)
> 
> Sorry, I meant the namespace bar

Added the namespace bar and updated the PR. PTAL. Thanks

**Comment by debsmita1** [comment]
verified all the changes on a dev cluster. Thank you @vikram-raj 

/lgtm

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**